mkdocs.yml

```diff
@@ -207,6 +207,7 @@ extra:
         kafkaversion: 2.5
         kafkarelease: 5.5.0-ccs
         ksqldbversion: 0.11.0
+        ksqldbcpversion: 6.1.0-beta200715032424
```

Comment:
Hey @JimGalasyn I couldn't discern the pattern regarding which variables are named `version` and which are named `release`. Should this new variable be named `release` instead?

Reply:
I was trying to follow the pattern from the main CP doc set: https://github.com/confluentinc/docs/blob/887c6bcf2ff152c23e836dd8f653da691a5247c3/conf.py#L214. I added the "ksqldb" variables to be specific to the Standalone distro, so I'm fine with this new variable name.